components/appointmentPicker/appointmentPickerController.js

```diff
@@ -102,7 +101,11 @@ define('appointmentPickerController', [], function() {
 				}
 				return ('No appointments available during the week of the selected date - FULL' + ($scope.appointmentPickerSharedProperties.isLoggedIn ? ' - overscheduled by ' + Math.abs(info.appointmentsAvailable) + ' appointments' : ''))
 			}
-		}
+		};
+
+		$scope.isVirtualAppointmentType = () => {
+			return $scope.appointmentType.includes('virtual-');
```

Comment:
I honestly think it could go either way. It seemed easier to me when writing this to just encode it in the lookupName, but I don't have a strong opinion one way or the other